frontend/main-app/src/components/DashboardAccordion/FormStatus.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "@digdir/design-system-react";
+import { DocPencilIcon } from "@navikt/aksel-icons";
+import { CustomParagraph } from "../CustomParagraph";
+import React from "react";
+
+interface FormStatusProps {
+  form: Form;
+}
+
+export const FormStatus = ({ form }: FormStatusProps) => {
```

Comment:
Here maybe we need to handle undefined status by adding:
` if (!form || !form.status) return ;`

Reply:
Yes, it will look nicer to not show the component at all, rather than showing just the headline 👍